orcid-web/src/main/resources/freemarker/includes/ng2_templates/id-banner-ng2-template.ftl

```diff
@@ -0,0 +1,38 @@
+<script type="text/ng-template" id="id-banner-ng2-template">
+    <div class="id-banner <#if inDelegationMode>delegation-mode</#if>"> 
+        <#if inDelegationMode><span class="delegation-mode-warning">${springMacroRequestContext.getMessage("delegate.managing_record")}</span></#if>
+        <!-- Name -->
+
+        <#if springMacroRequestContext.requestUri?contains("/my-orcid")>
```

Comment:
@lizkrznarich Hi! 
I am using this conditional to check if I am on `/my-orcid` or in the public version. Is this a good way, is there any corner case that might break this?